lib/service/logger.js

```diff
@@ -24,7 +24,7 @@ function _log(level, message) {
     level
   };
 
-  logMessage = JSON.stringify(logMessage) + '\n';
+  logMessage = `${JSON.stringify(logMessage)}\n`;
```

Comment:
interesting... for concatenating just two values (even if not already strings),  a + b and ``{$a}${b}`` behave identically (same gc activity), with addition running 10% faster.  The disparity above was when concatenating / interpolating multiple values.

Makes sense:  a + b does not need an intermediary string, but a + b + c + ... + z may generate O(n^2) temp strings (a + b), ((a + b) + c), etc.